File: Entidades/LocalidadEN.vb
Public Class LocalidadEN
    Private _codloc As Integer
    Public Property CodLoc() As Integer
        Get
            Return _codloc
        End Get
        Set(ByVal value As Integer)
            _codloc = value
        End Set
    End Property

    Private _codprovincia As Integer
    Public Property CodProvincia() As Integer
        Get
            Return _codprovincia
        End Get
        Set(ByVal value As Integer)
            _codprovincia = value
        End Set
    End Property


    Private _descripcion As String
    Public Property Descripcion() As String
        Get
            Return _descripcion
        End Get
        Set(ByVal value As String)
            _descripcion = value
        End Set
    End Property

    Private _codigopostal As String
    Public Property CodigoPostal() As String
        Get
            Return _codigopostal
        End Get
        Set(ByVal value As String)
            _codigopostal = value
        End Set
    End Property
End Class

File: Entidades/LocalidadEN.cs

namespace Entidades
{
    public class LocalidadEN
    {
        private int _codloc;

        public int CodLoc
        {
            get
            {
                return _codloc;
            }

            set
            {
                _codloc = value;
            }
        }

        private int _codprovincia;

        public int CodProvincia
        {
            get
            {
                return _codprovincia;
            }

            set
            {
                _codprovincia = value;
            }
        }

        private string _descripcion;

        public string Descripcion
        {
            get
            {
                return _descripcion;
            }

            set
            {
                _descripcion = value;
            }
        }

        private string _codigopostal;

        public string CodigoPostal
        {
            get
            {
                return _codigopostal;
            }

            set
            {
                _codigopostal = value;
            }
        }
    }
}